.vscode/launch.json

```diff
@@ -115,6 +115,15 @@
       "cwd": "${workspaceRoot}",
       "MIMode": "gdb",
       "preLaunchTask": "Build: DIM_test"
-    }
+    },
+    {
+      "name": "Debug Tests: Shared",
+      "type": "cppdbg",
+      "request": "launch",
+      "program": "${workspaceRoot}/build_x86/firmware/shared/shared_test",
+      "cwd": "${workspaceRoot}",
+      "MIMode": "gdb",
+      "preLaunchTask": "Build: shared_test"
+    },
   ]
 }
```

Comment:
In case you yerned for critique, add some new lines at the end of your files :neutral_face:

Reply:
Lol github doesn't seem to like this, does it actually matter?